catkin_ws/src/sensors/src/sensors_status.py

```diff
@@ -92,18 +92,15 @@ def __init__(self):
         super().__init__("IMU")
```

Comment:
this might be my fault (i don't remember) but self.current_reading and self.last_reading for IMU already starts with not None values, so the has_valid_data() function is pointless rn. It should start with None values. Lmk if you want to fix that or if I should

Reply:
yeah i just made whats there work with my changes i didnt want to refactor that whole file for this pr

Reply:
ok, I can branch off localization, fix this, and then open a PR to merge back into localization

Reply:
if its a small change just commit it directly to the branch

Reply:
if its a big change make a seperate PR to merge into noetic

Reply:
ok